public/app/features/canvas/runtime/element.tsx

```diff
@@ -82,6 +82,7 @@ export class ElementState implements LayerElement {
     const placement = this.options.placement ?? ({} as Placement);
 
     const style: React.CSSProperties = {
+      cursor: 'grab',
```

Comment:
Would it be possible to use the `grabbing` cursor when the mouse is actively clicked?

Also it might make sense to set the mouse back to just `pointer` when `inlineEditing` is disabled (see `scene.isEditingEnabled`)
![Screen Shot 2022-07-20 at 5 41 48 AM](https://user-images.githubusercontent.com/22381771/179974237-c27457ef-9668-4ab4-b1f3-f9a3f8f2e0b9.png)

Reply:
Yeah, was thinking about using grabbing as well, but didn't find a quick way to implement. Will look at it more.

Good idea, with inlineEditing disabled! Should be easy enough to implement.

Reply:
For `grabbing` cursor we may be able to tie into the `dragStart` and `dragEnd` events to enable / disable that css

Reply:
Done:
![Jul-21-2022 08-59-38](https://user-images.githubusercontent.com/60050885/180260116-22b5c9fd-18bb-40a7-9c2b-9b8c1533fc83.gif)